### ISSUE TITLE
Upgrade ajv to remove critical dependency warning

### ISSUE THREAD

**Issue by trungdq88** [josdejong/jsoneditor#414]
My project with webpack warns this:

```
Compiled with warnings.

./ssv-portals-node-modules/~/ajv/lib/async.js
96:20-33 Critical dependency: the request of a dependency is an expression

./ssv-portals-node-modules/~/ajv/lib/async.js
119:15-28 Critical dependency: the request of a dependency is an expression

./ssv-portals-node-modules/~/ajv/lib/compile/index.js
13:21-34 Critical dependency: the request of a dependency is an expression
```
The problem is jsoneditor is using ajv 4.x, the issue has been addressed in https://github.com/epoberezkin/ajv/issues/373 quite long ago. Should we make an update to ajv 5.x?

**Comment by josdejong** [comment]
v5 is just updated in the develop branch, and this issue should be fixed in the first next release.

See https://github.com/josdejong/jsoneditor/pull/412

**Comment by trungdq88** [comment]
Thank you! Is there any plan for the next release yet?

**Comment by josdejong** [comment]
I hope to do a release this weekend.

**Comment by trungdq88** [comment]
Great! Thank for your works.

**Comment by josdejong** [comment]
you're welcome :)

**Comment by josdejong** [comment]
I've just released `jsoneditor@5.7.1` containing the latest version of ajv (`5.2.0`). This should remove these warnings, if not I would be glad to hear.
